recipes/bdwgc/all/conanfile.py

```diff
@@ -78,24 +80,19 @@ def requirements(self):
             self.requires("libatomic_ops/7.6.10")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        os.rename("gc-{}".format(self.version), self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
 
     def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         self._cmake = CMake(self)
         for option, _ in self._autotools_options_defaults:
-                self._cmake.definitions["enable_{}".format(option)] = self.options.get_safe(option)
+            self._cmake.definitions["enable_{}".format(option)] = self.options.get_safe(option)
         self._cmake.definitions["disable_gc_debug"] = not self.options.gc_debug
         self._cmake.definitions["disable_handle_fork"] = not self.options.handle_fork
         self._cmake.definitions["install_headers"] = True
         self._cmake.definitions["build_tests"] = False
-        if tools.is_apple_os(self.settings.os):
```

Comment:
does conan's CMake helper handle that for you? I think it should be able to set `CMAKE_OSX_ARCHITECTURES` sometimes

Reply:
Yes, at least it's better handled by CMake helper.